docs/components/huggingface-embeddings/index.md

```diff
@@ -0,0 +1,25 @@
+# Get more control of your data with Custom Embeddings
+
+By default, AAQ uses
+[OpenAI text embeddings](https://platform.openai.com/docs/guides/embeddings/what-are-embeddings)
+to generate text vectors. But we give the option of deploying and hosting custom huggingface
+[embedding models](https://huggingface.co/spaces/mteb/leaderboard).
+
+---
+
+**Note**
+
+Huggingface has an extensive list of embeddings models, some lightweight and
+some heavy. Please carefully choose the model that will fit your use case best.
+
+---
+
+<div class="grid cards" markdown>
+
+- :octicons-gear-16:{ .lg .middle } **Deployment**
+
+  ***
+
+  Deploying the custom huggingface embeddings model and Configuring AAQ to use huggingface embeddings.
+
+  [:octicons-arrow-right-24: More info](./deployment.md)
```

Comment:
This is showing up broken!
![Screenshot 2024-08-14 at 1 49 12 PM](https://github.com/user-attachments/assets/caef3960-73d6-46ec-90fe-abbdfb5c1df1)

Reply:
It is showing up fine on my side. Can you share a screenshot of the full page?